pom.xml

```diff
@@ -1770,22 +1791,55 @@
                 <artifactId>accumulo-core</artifactId>
                 <version>${accumulo.version}</version>
                 <scope>provided</scope>
+                <exclusions>
+                    <!-- exclude hadoop 3+ only jars, since we're still compiling against haddop 2.8 -->
+                    <exclusion>
+                        <groupId>org.apache.hadoop</groupId>
+                        <artifactId>hadoop-client-api</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>org.apache.hadoop</groupId>
+                        <artifactId>hadoop-client-runtime</artifactId>
+                    </exclusion>
+                </exclusions>
             </dependency>
             <dependency>
                 <groupId>org.apache.accumulo</groupId>
                 <artifactId>accumulo-server-base</artifactId>
                 <version>${accumulo.version}</version>
                 <scope>provided</scope>
+                <exclusions>
+                    <!-- exclude hadoop 3+ only jars, since we're still compiling against haddop 2.8 -->
+                    <exclusion>
+                        <groupId>org.apache.hadoop</groupId>
+                        <artifactId>hadoop-client-api</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>org.apache.hadoop</groupId>
+                        <artifactId>hadoop-client-runtime</artifactId>
```

Comment:
Do we need to add them back in bundles and/or install scripts?

Do we need install-hadoop2 / install-hadoop3 scripts?

Reply:
i just left the install scripts as installing hadoop 2.8, and that seemed to work with a hadoop 3 environment... but you're right, it would probably be a good idea to modify the install scripts to add these if hadoop version > 3, as likely some user will update the version number. I'll make that change

Reply:
fixed